p2p/protocol/identify/id.go

```diff
@@ -428,9 +428,6 @@ func (ids *IDService) getSnapshot() *identifySnapshot {
 	if !ids.disableSignedPeerRecord {
 		if cab, ok := peerstore.GetCertifiedAddrBook(ids.Host.Peerstore()); ok {
 			snapshot.record = cab.GetPeerRecord(ids.Host.ID())
-			if snapshot.record == nil {
-				log.Errorf("latest peer record does not exist. identify message incomplete!")
```

Comment:
not worth a log when this occurs? seems at least debug worthy

Reply:
Yeah, maybe as a debug entry. However, it'll happen whenever we don't have addresses.

Reply:
You know, I'm going to be petty. It's not worth the extra indent I'd need to handle that case.

Reply:
Our debug logging is to verbose to be useful anyways.